app/views/images/index.html.haml

```diff
@@ -0,0 +1,14 @@
+- if @images.any?
+  - @images.each do |image|
+    %h2= image.name
+    = image_tag image.image.url(:thumb)
+    = link_to image.name, image_path(image)
+    = link_to "Add comment", new_image_comment_path(image)
+    - if image.comments.any?
+      %ul
+        - image.comments.each do |comment|
+          %li
+            = comment.comments
+- else
+  No images yet
+%a(href='images/new') Add an image
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))